### ISSUE TITLE
HTML generated code isn't up to standards.

### ISSUE THREAD

**Issue by alator21** [openSUSE/osem#1472]
HTML code generated by the app isnt up to any of the HTML standards.You can check this out by pasting the source code from any page of the project in the [w3 validator](https://validator.w3.org/).Is that okay?Should this be fixed?

**Comment by Ana06** [comment]
Yes, I would say that any improvementin the HTML is welcome! :wink:
